addons/mail/static/src/models/thread.js

```diff
@@ -2105,6 +2111,13 @@ registerModel({
         isDescriptionEditableByCurrentUser: attr({
             compute: '_computeIsDescriptionEditableByCurrentUser',
         }),
+        /**
+         * States whether the attachment file explorer should be opened
+         * automatically for this thread.
+         */
+        isFileBrowserOpen: attr({
+            default: false,
+        }),
```

Comment:
Confusing field name, and name does not match the documentation

This is just `hasAttachments`?

But do we really need a field for that, can't we just check attachment length in code?